Global.asax.cs

```diff
@@ -10,7 +10,7 @@ public class Global : System.Web.HttpApplication
         protected void Application_Start()
         {
             Microsoft.ApplicationInsights.Extensibility.TelemetryConfiguration
-                .Active.InstrumentationKey = ConfigurationManager.AppSettings["AppInsightKey"] ?? System.Environment.GetEnvironmentVariable("AppInsightKey");
+                .Active.InstrumentationKey = ConfigurationManager.AppSettings["AppInsightKey"] ?? System.Environment.GetEnvironmentVariable("AppInsightKey") ?? string.Empty;
```

Comment:
I had that for local development because I know if I put it in web.config I'll end up committing it by mistake. The fall back to empty string is because AppInsight throws it's set to null